backend/controllers/appointments/prepostAppoinments.js

```diff
@@ -187,11 +185,9 @@ const prepostAppointmentsFactory = ({
         },
       })
     } catch (error) {
-      logError(req.originalUrl, error, serviceUnavailableMessage)
-      res.status(500)
-      res.render('error.njk', {
-        url: authSource === 'nomis' ? `/offenders/${offenderNo}/add-appointment` : '/videolink/prisoner-search',
-      })
+      res.locals.redirectUrl =
+        authSource === 'nomis' ? `/offenders/${offenderNo}/add-appointment` : '/videolink/prisoner-search'
```

Comment:
don't think you'd need this as this would redirect a prison user to the court user's service

Reply:
I think it might be worth creating a separate PR to remove these.